arango/formatter.py

```diff
@@ -249,20 +249,9 @@ def format_collection(body: Json) -> Json:
         result["min_revision"] = body["minRevision"]
     if "schema" in body:
         result["schema"] = body["schema"]
+    if "computedValues" in body:
+        result["computedValues"] = body["computedValues"]
 
-    # New in 3.10
-    if body.get("computedValues") is not None:
-        result["computedValues"] = [
-            {
-                "name": cv["name"],
-                "expression": cv["expression"],
-                "overwrite": cv["overwrite"],
-                "computedOn": cv["computedOn"],
-                "keepNull": cv["keepNull"],
-                "failOnWarning": cv["failOnWarning"],
-            }
-            for cv in body["computedValues"]
-        ]
```

Comment:
Need to revisit this as I am aware of https://github.com/ArangoDB-Community/python-arango/pull/247#discussion_r1200342072

Building `result["computedValues"]` via a list comprehension will cause `KeyError`s if some optional computedValue properties have not been specified (e.g `computeOn`, `keepNull`, `failOnWarning`)

Reply:
It seems like https://github.com/ArangoDB-Community/python-arango/pull/224 was introduced as a consequent of using a list comprehension for `computedValues`

Does it not make sense to just follow how the [schema property is set](https://github.com/ArangoDB-Community/python-arango/blob/main/arango/formatter.py#L250-L251)?

Reply:
This should be adapted accordingly, especially since the current implementation is prone to exceptions, as you pointed out.

I think it makes perfect sense to follow how the schema property is set. Maybe we should keep the _is not None_, such that _result["computedValues"]_ stays non-null. I imagine something like this:
```python
if body.get("computedValues") is not None:
    result["computedValues"] = body["computedValues"]
```

This should give us the equivalent, without raising _KeyError_. I am not adamant about using _not None_ instead of _"computedValues" in body_, but I am suggesting the former because it does not change the function's behavior.

Reply:
understood, and I agree that it's probably best not to change the existing behaviour

added in 3acd627